client/app/dedicatedUniverseComponents/bandwidth/bandwidth.js

```diff
@@ -1,9 +1,6 @@
-/**
- * @type filter
- * @name filters:bandwidth
- */
+import angular from 'angular';
```

Comment:
Could be renamed as `client/app/dedicatedUniverseComponents/bandwidth/bandwidth.filter.js`